src/app/child-dev-project/children/child-details/child-details.component.html

```diff
@@ -470,131 +431,10 @@ <h3>Coaching Attendance</h3>
-            matInput
-            formControlName="health_bloodGroup"
-            placeholder="Blood Group"
-            type="text"
-          />
-        </mat-form-field>
-      </div>
-
-      <div>
-        <mat-form-field>
-          <input
-            matInput
-            formControlName="health_lastDentalCheckup"
-            placeholder="Last Dental Check-Up"
-            [matDatepicker]="lastDentalCheckupDatepicker"
-            [disabled]="!editing"
-          />
-          <mat-datepicker-toggle
-            matSuffix
-            [for]="lastDentalCheckupDatepicker"
-          ></mat-datepicker-toggle>
-          <mat-datepicker #lastDentalCheckupDatepicker></mat-datepicker>
-        </mat-form-field>
-
-        <mat-form-field>
-          <input
-            matInput
-            formControlName="health_lastEyeCheckup"
-            placeholder="Last Eye Check-Up"
-            [matDatepicker]="lastEyeCheckupDatepicker"
-            [disabled]="!editing"
-          />
-          <mat-datepicker-toggle
-            matSuffix
-            [for]="lastEyeCheckupDatepicker"
-          ></mat-datepicker-toggle>
-          <mat-datepicker #lastEyeCheckupDatepicker></mat-datepicker>
-        </mat-form-field>
-
-        <mat-form-field>
-          <input
-            matInput
-            formControlName="health_lastENTCheckup"
-            placeholder="Last ENT Check-Up"
-            [matDatepicker]="lastENTCheckupDatepicker"
-            [disabled]="!editing"
-          />
-          <mat-datepicker-toggle
-            matSuffix
-            [for]="lastENTCheckupDatepicker"
-          ></mat-datepicker-toggle>
-          <mat-datepicker #lastENTCheckupDatepicker></mat-datepicker>
-        </mat-form-field>
-
-        <mat-form-field>
-          <input
-            matInput
-            formControlName="health_lastVitaminD"
-            placeholder="Last Vitamin D"
-            [matDatepicker]="lastVitaminDDatepicker"
-            [disabled]="!editing"
-          />
-          <mat-datepicker-toggle
-            matSuffix
-            [for]="lastVitaminDDatepicker"
-          ></mat-datepicker-toggle>
-          <mat-datepicker #lastVitaminDDatepicker></mat-datepicker>
-        </mat-form-field>
-        <mat-form-field>
-          <input
-            matInput
-            formControlName="health_lastDeworming"
-            placeholder="Last De-Worming"
-            [matDatepicker]="lastDewormingDatepicker"
-            [disabled]="!editing"
-          />
-          <mat-datepicker-toggle
-            matSuffix
-            [for]="lastDewormingDatepicker"
-          ></mat-datepicker-toggle>
-          <mat-datepicker #lastDewormingDatepicker></mat-datepicker>
-        </mat-form-field>
-      </div>
-
-      <div>
-        <h3>Height & Weight Tracking</h3>
-        <app-health-checkup></app-health-checkup>
-      </div>
+      <app-health-form [child]="child"></app-health-form>
+      <br/>
+      <h3>Height & Weight Tracking</h3>
```

Comment:
should we move that part into the `app-health-form` in order to get a single component for the whole section/panel?

Reply:
I think its alright to have multiple things displayed in one subfolder. We have the same in the education section.